README.md

```diff
@@ -34,6 +34,63 @@ Navigate to your application, click **Import End Devices**, select **The Things
 $ ttn-lw-cli end-devices create --application-id test-app < devices.json
 ```
 
+## The Things Network Stack V2
+
+### Configuration
+
+Configure with environment variables, or command-line arguments. See `--help` for more details:
+
+```bash
+$ export TTN_APP_ID="my-ttn-app"                    # TTN App ID
+$ export TTN_APP_ACCESS_KEY="ttn-account-v2.a..."   # TTN App Access Key (needs `devices` permissions)
+$ export FREQUENCY_PLAN_ID="EU_863_870_TTN"         # Frequency Plan for exported devices
+```
+
+See [Frequency Plans](https://thethingsstack.io/reference/frequency-plans/) for the list of frequency plans available on The Things Stack. For example, to use `United States 902-928 MHz, FSB 1`, you need to specify the `US_902_928_FSB_1` frequency plan ID.
+
+Private TTN V2 deployments are also supported, and require extra configuration. See `ttn-lw-migrate device --help` for more details. For example, to override the discovery server address:
+
+```bash
+$ export TTN_DISCOVERY_SERVER_ADDRESS="discovery.thethings.network:1900"
+```
+
+### Export Devices
+
+To export a single device using its Device ID (e.g. `mydevice`):
+
+```bash
+$ ttn-lw-migrate device --source ttn "mydevice" > devices.json
+```
+
+In order to export a large number of devices, create a file named `device_ids.txt` with one device ID per line:
+
+```
+mydevice
+otherdevice
+device3
+device4
+device5
+```
+
+And then export with:
+
+```bash
+$ ttn-lw-migrate device --source ttn < device_ids.txt > devices.json
+```
+
+### Export Applications
+
+Similarly, to export all devices of application `my-app-id`:
+
+```bash
+$ ttn-lw-migrate application --source ttn "my-app-id" > devices.json
+```
+
+### Notes
+
+- Payload formatters are not exported. See [Payload Formatters](https://thethingsstack.io/integrations/payload-formatters/).
+- Exporting a large number of devices may take a long time. If you plan on having the devices rejoin on The Things Stack, you can speed up the process by setting the `--ttn.without-frame-counters` flag.
```

Comment:
I still don't understand why we scope this to frame counters, and not the session as a whole.

Also, it can be just `--ttn.frame-counters=false`, if the attribute's default value is `true`.

Reply:
- I don't see this flag getting used in the code.
- Let's indeed make this about the entire session.
- I would strongly suggest NOT migrating sessions by default.